### PR TITLE
fix for backbutton flag

### PR DESCRIPTION
**Pull request by danielknobe**
Added missing flag (on screen backbutton) input implementation.

### CODE REVIEW

File: src/Global.h [hunk 1]
```diff
@@ -22,6 +22,10 @@ Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 
 #include <string>
 
+#if (defined __APPLE__)
```
Comment (ngc92) [inline src/Global.h:25]: Is this change related to the others?

**Comment by ngc92** [comment]
Nice cleanup. Less code, but now it actually works :)
I'm not sure if the current placement of the button is good, though. This way, the ball regularly passes behind the flag.

File: src/Global.h (hunk 1 above)
Comment (danielknobe) [reply src/Global.h:25]: Yes :). The macros in global.h may not work correctly. The main change made in this PR ist a good example for that.

**Comment by danielknobe** [comment]
This ist a good point! I will add it to the PR.

**Comment by danielknobe** [comment]
I reduced the size of the flag but made the button area a little bit bigger. I think the flag no longer covers much of the ball. I think this is a good solution.
